requirements/requirements-all.txt

```diff
@@ -11,3 +11,4 @@ pyyaml>=3.12
 rich>=9.10.0
 ubiquerg>=0.5.2
 yacman>=0.9.2
+pydantic-argparse==0.8.0
```

Comment:
Why fix the version rather than just lower bounding?

Reply:
Fixed in https://github.com/pepkit/looper/pull/440/commits/17cf965bfc08b23914af3210096a7dea121383a7.